sections/install.md

```diff
@@ -64,7 +64,7 @@ repos:
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
 -   repo: https://github.com/psf/black
-    rev: 21.12b0
+    rev: 22.12.0
```

Comment:
can you set it to an intentionally out-of-date version such that others are not motivated to send spammy bump PRs